symfony/webpack-encore-pack/1.0/webpack.config.js

```diff
@@ -1,31 +1,35 @@
 var Encore = require('@symfony/webpack-encore');
 
 Encore
-    // the project directory where compiled assets will be stored
+    // directory where compiled assets will be stored
     .setOutputPath('public/build/')
-    // the public path used by the web server to access the previous directory
+    // public path used by the web server to access the output path
     .setPublicPath('/build')
-    // the public path you will use in Symfony's asset() function - e.g. asset('build/some_file.js')
-    .setManifestKeyPrefix('build/')
+    // public path you will use in Symfony's asset() function - e.g. asset('build/some_file.js')
+    // you don't need to set this unless you deploy to a subdirectory or use a CDN
+    //.setManifestKeyPrefix('build/')
+
+    // add 1 entry for each "page" of your app
+    // (including one that's included on every page - e.g. "app")
+    .addEntry('app', './assets/js/app.js')
+    //.addEntry('page1', './assets/js/page1.js')
+    //.addEntry('page2', './assets/js/page2.js')
 
     .cleanupOutputBeforeBuild()
     .enableSourceMaps(!Encore.isProduction())
-
-    // the following line enables hashed filenames (e.g. app.abc123.css)
+    // enables hashed filenames (e.g. app.abc123.css)
     .enableVersioning(Encore.isProduction())
 
-    // uncomment to define the assets of the project
-    //.addEntry('js/app', './assets/js/app.js')
-    //.addStyleEntry('css/app', './assets/css/app.scss')
```

Comment:
As you added a default JS entry, why not add one for CSS too?

Reply:
`addStyleEntry` is kind of an anti-pattern. So, I doc'ed it, but people shouldn't really need to use it, if they want to do things correctly.